insights/specs/insights_archive.py

```diff
@@ -80,6 +80,7 @@ class InsightsArchiveSpecs(Specs):
     getconf_page_size = simple_file("insights_commands/getconf_PAGE_SIZE")
     getenforce = simple_file("insights_commands/getenforce")
     getsebool = simple_file("insights_commands/getsebool_-a")
+    grubenv = first_file(["/boot/grub2/grubenv", "/boot/efi/EFI/redhat/grubenv"])
```

Comment:
Since the spec is changed to command, here the spec should be updated as:
```
grubenv = first_file(["insights_commands/grub2-editenv_list", "/boot/grub2/grubenv", "/boot/efi/EFI/redhat/grubenv"])
```